`debug-to-string.js` requires `@glimmer/env`, but does not depend on it

When attempting to update ember-template-lint to use 0.47.6, we get the following error:

```
    Cannot find module '@glimmer/env' from 'debug-to-string.js'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:221:17)
      at Object.<anonymous> (node_modules/@glimmer/util/dist/commonjs/es5/lib/debug-to-string.js:7:12)
```

See https://github.com/ember-template-lint/ember-template-lint/pull/1110

--- a/packages/@glimmer/util/package.json
+++ b/packages/@glimmer/util/package.json
@@ -5,6 +5,7 @@
   "repository": "https://github.com/tildeio/glimmer/tree/master/packages/@glimmer/util",
   "license": "MIT",
   "dependencies": {
+    "@glimmer/env": "0.1.7",
     "@simple-dom/interface": "^1.4.0"
   },
   "devDependencies": {

